tools/scripts/setup_cygwin.sh

```diff
@@ -58,7 +58,7 @@ install_cmake() {
 setup_ewdk() {
     if [[ ! -d /cygdrive/c/ewdk ]]; then
         echo "Fetching EWDK. Please wait..."
-        wget -nv -O /tmp/ewdk.zip "https://go.microsoft.com/fwlink/p/?LinkID=699461"
+        curl "https://go.microsoft.com/fwlink/p/?LinkID=699461" -L --output /tmp/ewdk.zip
```

Comment:
imo, something like local dl_path=$(mktemp) is the preferred approach over using hardcoded tmp paths... (or mktemp -d with the temp files within, that you can remove when finished).